addons/common/functions/fnc_checkFiles.sqf

```diff
@@ -52,23 +52,25 @@ _addons = _addons select {_x find "ace_" == 0};
 ///////////////
 // check dlls
 ///////////////
-{
-    private _versionEx = _x callExtension "version";
+if (productVersion select 6 == "Windows") then {
```

Comment:
That you rather check for Linux/OSX and skip the test then to check for "Windows". (It's just about the strings). My fear is that BI could change the string from `Windows` to something different (e.g. `Win`, `Windows10`, `Windows-x86`). And I would like to see it rather fail for the least amount of players if that will ever happen.